wordpress-mobile/WordPress-Aztec-iOS
# Update toolbar with grid icons and adding source code item.

How to test:

 - Check if the icons on the toolbar have the gridicons look
 - Check if the source code button on the toolbar works correctly.
